avg of 10 numbers(array)

```diff
@@ -0,0 +1,19 @@
+#include<stdio.h>
+#include<conio.h>
+int main()
+{
+  int a[10],i,sum = 0;
+  float avg;
+  printf("enter 10 numbers");
+  for(i= 0;i< = 9;i++)
```

Comment:
There shouldn't be any space while using a conditional operator. Replace i< = 9 with i<=9.

Reply:
@anchal071 Please make the changes suggested by @Anjali-1801.

Reply:
sir ; I  also want to make changes suggested by her but I have no idea that how to do that.

Reply:
Make the changes in your cloned repository (in local device) and then push the changes, it'll reflect in your PR.